modules/manage/pages/manage-buckets/migrate-bucket.adoc

```diff
@@ -58,12 +81,104 @@ include::manage:example$migrate-bucket-storage-backend.sh[tag=recover-node]
 ----
 include::manage:example$migrate-bucket-storage-backend.sh[tag=rebalance-cluster]
 ----
-. After triggering each node to rewrite its vBuckets, verify the node is now using the new storage backend. Re-run the command from step 2 to list the nodes and any storage backend overrides:
+. After triggering each node to rewrite its vBuckets, verify the node is now using the new storage backend. 
+Re-run the command from step 2 to list the nodes and any storage backend overrides:
 +
 [source,console]
 ----
 include::manage:example$storage-backend-override-node3.sh[]
 ----
 +
-The `null` under node3 indicates that it does not have a storage backend override. It has migrated to the new storage backend.
+The `null` under node3 indicates that it does not have a storage backend override. 
+It has migrated to the new storage backend.
 . Repeat the previous two steps for the remaining nodes in the cluster.
+
+[#disk_usage]
+== Disk Use Under Couchstore Verses Magma
+
+If you migrate a bucket's storage from Couchstore to Magma, you may see increased disk usage. 
+Couchstore's default threshold for fragmentation is 30%.
+When a Couchstore bucket reaches this threshold, Couchbase Server attempts to fully compact the bucket. 
+If the bucket has a low write workload, Couchbase Server may be able to compact the bucket to 0% fragmentation.
+
+Magma's default fragmentation threshold is 50%.
+Couchbase Server treats this threshold differently than the Couchstore threshold.
+It does not try to compact the bucket until it has 0% fragmentation.
+Instead, Couchbase Server tries to compact a Magma bucket to maintain its fragmentation at 50%.
+This maintenance of 50% fragmentation may result in greater disk use for a Magma-backed bucket verses the Couchstore-backed bucket.
+
+If you find that a bucket migrated to Magma has high disk use, you have two options:
+
+* Reduce the fragmentation threshold of the Magma bucket. 
+FOr example, you could choose to reduce the fragmentation threshold to 30%.
+You should only consider changing the threshold if the bucket's workload is not write intensive.
+
+* Roll back the migration. 
+You can revert a bucket from Magma back to Couchstore after (or even during) a migration.
+See the next section for more information.
+
+== Rolling Back a Migration
+
+As you migrate each node's vBuckets to a new storage backend, you may decided that the migration is not meeting your needs.
+For example, you may see increased disk usage when moving from Couchstore to Magma as explained in xref:#disk_usage[Disk Use Under Couchstore Verses Magma].
+You can roll back the migration by changing the bucket's backend setting to its original value.
+Then have any migrated nodes rewrite their vBuckets back to the old backend.
+You can perform a roll back on an ongoing migration. 
```

Comment:
Personally reading these steps is a bit confusing. Could we rephrase these, by bullet pointing or numbers the steps to be performed or whatever you deem is more appropriate? 

About: 
> You can perform a roll back on an ongoing migration.

We seem to have mentioned the same thing a few lines above.
> You can revert a bucket from Magma back to Couchstore after (or even during) a migration.